commit 6143a4525c3e97bb56b6e5ccd09c7b89f648873a
Author: Vasily Gorbik <gor@linux.ibm.com>
Date:   Mon Jun 25 14:30:42 2018 +0200

    s390/scm_blk: correct numa_node in scm_blk_dev_setup
    
    [ Upstream commit d642d6262f4fcfa5d200ec6e218c17f0c15b3390 ]
    
    The numa_node field of the tag_set struct has to be explicitly
    initialized, otherwise it stays as 0, which is a valid numa node id and
    cause memory allocation failure if node 0 is offline.
    
    Acked-by: Sebastian Ott <sebott@linux.ibm.com>
    Signed-off-by: Vasily Gorbik <gor@linux.ibm.com>
    Signed-off-by: Martin Schwidefsky <schwidefsky@de.ibm.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/s390/block/scm_blk.c b/drivers/s390/block/scm_blk.c
index b1fcb76dd272..98f66b7b6794 100644
--- a/drivers/s390/block/scm_blk.c
+++ b/drivers/s390/block/scm_blk.c
@@ -455,6 +455,7 @@ int scm_blk_dev_setup(struct scm_blk_dev *bdev, struct scm_device *scmdev)
 	bdev->tag_set.nr_hw_queues = nr_requests;
 	bdev->tag_set.queue_depth = nr_requests_per_io * nr_requests;
 	bdev->tag_set.flags = BLK_MQ_F_SHOULD_MERGE;
+	bdev->tag_set.numa_node = NUMA_NO_NODE;
 
 	ret = blk_mq_alloc_tag_set(&bdev->tag_set);
 	if (ret)
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1147_6143a4525c3e97bb56b6e5ccd09c7b89f648873a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1147_6143a4525c3e97bb56b6e5ccd09c7b89f648873a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
